student-preferred-name-portlet-api/src/main/java/edu/wisc/portlet/preferred/form/PreferredNameExtended.java

```diff
@@ -3,6 +3,17 @@
 import org.apache.commons.lang.StringUtils;
 
 public class PreferredNameExtended extends PreferredName {
+  
+    public PreferredNameExtended() {
+      super();
+    }
+  
+    public PreferredNameExtended (PreferredName pn, String legalSirName) {
```

Comment:
Did `legalSirName` mean to be [`legalSurName`](http://en.wikipedia.org/wiki/Surname)?